cli/__init__.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# PYTHON_ARGCOMPLETE_OK
+
+"""
+An installer for Foreman.
+"""
+import os
+import obsah
+import sys
+from importlib import resources
+
+
+class ApplicationConfig(obsah.ApplicationConfig):
+    """
+    A class describing the where to find various files
+    """
+
+    @staticmethod
+    def name():
+        """
+        Return the name as shown to the user in the ArgumentParser
+        """
+        return 'rop'
+
+    @staticmethod
+    def data_path():
+        path = os.environ.get('ROP_DATA')
+        if path is None:
+            with resources.path(__name__, 'data') as fspath:
+                path = fspath.as_posix()
+
+        return path
+
+    @staticmethod
+    def inventory_path():
+        """
+        Return the inventory path
+        """
+        return os.environ.get('OBSAH_INVENTORY', os.path.join(os.getcwd(), 'inventories'))
+
+
+def main(cliargs=['--help'], application_config=ApplicationConfig):  # pylint: disable=R0914
+    """
+    Main command
+    """
+    if len(sys.argv) > 1:
+        cliargs = sys.argv[1:]
+
+    obsah.main(cliargs=cliargs, application_config=application_config)
```

Comment:
```suggestion
def main(cliargs=None, application_config=ApplicationConfig):  # pylint: disable=R0914
    """
    Main command
    """

    obsah.main(cliargs=cliargs, application_config=application_config)
```

https://github.com/theforeman/obsah/commit/583442c65a87f998d06d069653fe5a78e6661c21 is in

Reply:
Weird, I thought I had already put this back.